stingray/modeling/posterior.py

```diff
@@ -126,7 +131,12 @@ def logprior(t0, neg=False):
         # correctly!
         for pname in lpost.model.param_names:
             if not lpost.model.fixed[pname]:
-                logp += np.log(priors[pname](t0[ii]))
+                with warnings.catch_warnings(record=True) as out:
+                    logp += np.log(priors[pname](t0[ii]))
+                    if len(out) > 0:
```

Comment:
`out` should not be defined outside the `with` statement, so it should definitely be within the `with` statement.